phoenicis-library/src/main/java/org/phoenicis/library/LibraryManager.java

```diff
@@ -61,19 +61,19 @@ public LibraryManager(String shortcutDirectory) {
         for (File file : directoryContent) {
             if ("shortcut".equals(FilenameUtils.getExtension(file.getName()))) {
                 ShortcutDTO shortcut = fetchShortcutDTO(shortcutDirectoryFile, file);
-                String categoryName = shortcut.getCategory();
-                if (!categoryMap.containsKey(categoryName)) {
-                    categoryMap.put(categoryName, new ArrayList<>());
+                String categoryId = shortcut.getCategory();
+                if (!categoryMap.containsKey(categoryId)) {
+                    categoryMap.put(categoryId, new ArrayList<>());
                 }
-                categoryMap.get(categoryName).add(shortcut);
+                categoryMap.get(categoryId).add(shortcut);
```

Comment:
This looks like a simple grouping task, right?
I believe it would be better readable and shorter if you use the java streaming api for this. (see here: https://www.leveluplunch.com/java/examples/java-util-stream-groupingBy-example/)